test/autorest/generated/lrogroup/pollers.go

```diff
@@ -104,6 +102,12 @@ func httpPollerPollUntilDone(ctx context.Context, p HTTPPoller, frequency time.D
 	return p.FinalResponse(ctx)
 }
 
+func httpPollerHandleResponse(resp *azcore.Response) (*HTTPResponse, error) {
```

Comment:
Since these are specific to the poller type I was thinking these should be methods on the matching pollers.  It's all internal code though so it doesn't make much difference.

Reply:
Done